feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -102,68 +113,118 @@ func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes) *oc.Interface {
 	return i
 }
 
+// configureNetworkInstance creates nonDefaultVRF
+func configureNetworkInstance(t *testing.T, dut *ondatra.DUTDevice) {
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(nonDefaultVRF)
+	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(nonDefaultVRF).Config(), ni)
+
+	if *deviations.ExplicitGRIBIUnderNetworkInstance {
+		fptest.EnableGRIBIUnderNetworkInstance(t, dut, nonDefaultVRF)
+		fptest.EnableGRIBIUnderNetworkInstance(t, dut, *deviations.DefaultNetworkInstance)
+	}
+
+	dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
+	gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
+	// configure PBF in DEFAULT vrf
+	defNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
```

Comment:
I think configuring PBR is not part of the work/logic of configuring the VRF-1, therefore I think this part of the logic should be outside of this func configureNetworkInstance().

Reply:
@xw-g , I have used this function not just for VRF-1 configuration . I have moved all network-instance related to configurations to this function for both VRF-1 and DEFAULT network-instances.

Reply:
PBR is a thing that attached to an interface, not attached to a network instance. Therefore, I don't think it should be part of the network instance configuration.

Reply:
Actually, you are right. It was my bad, I messed up the policy schema.